GOVERNANCE.md

```diff
@@ -0,0 +1,246 @@
+* All meetings are logged and available at [`community-council/previous_meetings/`](https://github.com/sktime/community-council/tree/master/previous_meetings).
+* To contact the CC directly, please send an email to info@sktime.org. 
+
+## Decision making
+
+The purpose of this section is to formalize the decision-making process used by the sktime project. We clarify 
+* what types of changes we make decision on, 
+* how decisions are made, and 
+* who participates in the decision making.
+
+sktime's decision-making process is designed to take into account feedback from all community members and strives to find consensus, while avoiding deadlocks when no consensus can be found.
+
+All discussion and votes takes place on the project’s [issue tracker](https://github.com/alan-turing-institute/sktime/issues), [pull requests](https://github.com/alan-turing-institute/sktime/pulls) or [enhancement proposals](#sktime-enhancement-proposals). Some sensitive discussions and appointment votes occur on private chats.
+
+The CC reserves the right to overrule decisions.
+
+We distinguish between the following types of proposed changes. The corresponding decision making process is described in more detail below.  
+
+| Type of change | Decision making process | 
+|---|---|
+| Code additions, such as new algorithms | Lazy consensus, supported by the [algorithm inclusion criteria](#Algorithm-inclusion-criteria) |
+| Minor documentation changes, such as typo fixes, or addition/correction of a sentence | Lazy consensus | 
+| Code changes and major documentation changes | Lazy consensus | 
+| Changes to the API design, hard dependencies, or supported versions | Lazy consensus based on an [enhancement proposal](#sktime-enhancement-proposals) | 
+| Changes to sktime's governance (this document and the CoC) | Lazy consensus based on an [enhancement proposal](#sktime-enhancement-proposals) | 
+| Appointment | Voting | 
+
+### Stage 1: lazy consensus with veto right
+sktime uses a "consensus seeking" process for making decisions. The community tries to find a resolution that has no open objections among core developers. 
+
+* To accept proposed changes, we require approval by one core developer (lazy consensus) and no rejection by a core developer (veto right). 
+* Approvals and rejections can be expressed as +1 and -1 comments, respectively. 
+* Core developers are expected to give reasonable time to others to give their opinion on the pull request if they’re not confident others would agree.
+* More important changes that impact the full project require a more detailed analysis and a consensus that is both explicit and informed. These changes require an [enhancement proposal](#sktime-enhancement-proposals). 
+
+When no consensus can be found, the decision is escaled to [Stage 2: voting](#Stage-2:-voting).
+
+### Stage 2: voting 
+When no consensus can be found, any core developer can call for a vote at any point during the discussion. 
```

Comment:
yes, this change to 2 stages makes sense. Good we had the Git(hub) flow discussion to inspire this mechanism :-)